doc/source/whatsnew/v0.17.0.txt

```diff
@@ -226,11 +226,13 @@ or it can return False if broadcasting can not be done:
 
 Other API Changes
 ^^^^^^^^^^^^^^^^^
+
 - Enable writing Excel files in :ref:`memory <_io.excel_writing_buffer>` using StringIO/BytesIO (:issue:`7074`)
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 - Allow passing `kwargs` to the interpolation methods (:issue:`10378`).
 - Serialize metadata properties of subclasses of pandas objects (:issue:`10553`).
-
+- Boolean comparisons of a ``Series`` vs ``None`` will now be equivalent to comparing with ``np.nan``, rather than raise ``TypeError``, xref (:issue:`1079`).
+- Remove use of some deprecated numpy comparisons (:issue:`10569`)
```

Comment:
So I would keep the first one in the API changes section (but maybe then give an example to make it more clear)